frontend/src/MainPageGallery/MainPageMovieCard.css

```diff
@@ -12,22 +12,52 @@ button {
 
 .movie-card img {
     width: 180px;
+    border: none;
 }
 
 .movie-card {
-    margin: 1.3rem;
-    padding: 1.4rem 0;
-    width: 17rem;
+    margin-bottom: 3rem;
+    width: 250px;
+    height: 370px;
     font-weight: 200;
     /*background-color: #7FB5FF;*/
-    background-color: #F6F6F6;
+    background-color: #C4DDFF;
     border-radius: 8px;
```

Comment:
8px === 1rem  at least in most cases.

Btw: The unwritten standard for border radius is 5px as seen for example here on github: 
![grafik](https://user-images.githubusercontent.com/12148746/181012661-d267a037-7812-4b78-8863-a403da0c3b86.png)